packages/discovery-provider/src/queries/search_es.py

```diff
@@ -364,13 +364,16 @@ def personalize_dsl(dsl, current_user_id, must_saved):
 def default_function_score(dsl, ranking_field):
     return {
         "query": {
-            "script_score": {
+            "function_score": {
                 "query": {"bool": dsl},
-                "script": {
-                    "source": f"_score * Math.log(Math.max(doc['{ranking_field}'].value, 0) + 2)"
+                "field_value_factor": {
+                    "field": ranking_field,
+                    "factor": 20,
+                    "modifier": "ln2p",
                 },
+                "boost_mode": "multiply",
```

Comment:
does this mean boosts below 1 mean it's less boosted?

Reply:
yup but ln2p will do ln(follower_count + 2) which should never be below 1